### DISCUSSION TITLE
Show Media Release Status/Date on Title cards in Request List

### DISCUSSION THREAD

**Discussion by ourcontact** [sct/overseerr#1461] [Feature Requests]
I use Overseerr to manage our families wish list, often requesting movies months in advance.
Overseerr could be easier to use identifying titles on a request (wish) list as they become available.

I would like to suggest that the Media Status & Media Release date be shown on the summary title cards when reviewing the list of Requested titles
This information is already available but has to be checked by opening the title card and revealing detail

It would useful to add sort Requests by "Release Date" in addition to the options "Request Date" and "Last Modified"

The notification options would also be useful to include "Media Released" when the Media Status changes, for example from Status "Post Production" to "Released"

**Comment by phlurblepoot** [comment]
I second this, mostly so they can stop asking when I will add a movie. If they could just see the physical release date right there it would stop them asking.

**Comment by EricH9958** [comment]
Agreed, this would be a great feature.
